include/game.h

```diff
@@ -775,22 +775,26 @@ typedef struct {
     /* 0x80 */ u8 Icon[3][128];
 } MemcardHeader; /* size=0x200 */
 
+typedef struct {
```

Comment:
How did you figure out this is a separate struct? Not seeing a usage in the new function